src/components/migrationToV3.stories.mdx

```diff
@@ -83,6 +86,154 @@ const MyApp = () => {
     <button onClick={joinMeeting}>Join Meeting</button>;
   );
 };
+
+### Meeting Provider prop change
+Meeting provider no longer takes the following props:
+- `MeetingSessionConfiguration` props:
+  - `VideoDownlinkBandwidthPolicy`
+  - `VideoUplinkBandwidthPolicy`
+  - `simulcastEnabled`
+  - `reconnectTimeoutMs`
+  - `keepLastFrameWhenPaused`
+- Other props:
+  - `logLevel`
+  - `postLogConfig`
+  - `logger`
+  - `enableWebAudio`
+  - `activeSpeakerPolicy`
+
+These properties can rather be updated by passing the `MeetingSessionConfiguration` object or by passing optional parameters to `meetingManager.join()` method.
```

Comment:
It's better to explicitly call out the parameter of `join` method here. It's the migration guide, it's ideal that the builder can finish the migration without checking other docs/API definition.

* passing the `MeetingSessionConfiguration` object via `meetingSessionConfiguration` parameter
* passing optional parameters via `options` parameter